### ISSUE TITLE
Optimizer removes 'Jr.' and 'Sr.' from player names in lineups output generated

### ISSUE THREAD

**Issue by leodigi** [DimaKudosh/pydfs-lineup-optimizer#194]
This is not a critical issue, however it is causing some issue with the post processing code that I am writing to manage which lineups I want to run in specific contests based on certain criteria by which I am analyzing each lineup. I am having to either "find and replace all" in exported csv first before running my analyzer or write alternative code to search in each dict key's value vs just seeing if value is in dict.values() which has to be an exact match.

**Comment by sansbacon** [comment]
It does raise a good issue - at least on DK, there is no good reason to split into first and last name and then reassemble into full name. The DK-supplied files (salaries and entries) both use a single name. I wonder if this is an artifact of supporting other sites?

One possible workaround is to subclass Player and replace full_name property with a full_name parameter that you pass to the constructor (as a keyword, not a positional argument). If you are creating your own player objects and then loading them into the optimizer then this should be (relatively) straightforward to do. If you are using the CSV loader classes, it may be more complicated.

This is a starting point for how you might do it - has not been tested so this is not plug and play, just an illustration of the concept.

```
class FullNamePlayer(Player):
    
    def __init__(self, *a, **k):
        if 'full_name' in k:
            full_name = k.pop('W')
        super().__init__(self, *a, **k)
        self._full_name = full_name

    @Player.full_name.getter
     def full_name(self):
         return self._full_name
```

**Comment by DimaKudosh** [comment]
Fixed in version 3.5.0
